style.css

```diff
@@ -0,0 +1,13 @@
+body{
+    width:80%;
+    display:box;
+    flex-direction:column;
+    padding:0 auto;
+    padding-left: 10%;
+    justify-content:center;
+
+    /* background: black; */
+}
+#title{
```

Comment:
Please give one space indentation after the CSS selector ( as an example here give one space indentation after 'body' and '#title'). 
Use 2 space indentations before any property and one space indentation after the semicolon ( as an example here give 2 space indentations before 'width:80%' 'display:box' and 1 space after the colon).
 It will make your CSS code cleaner and more organized.